packages/components/src/color-picker/component.tsx

```diff
@@ -59,32 +57,23 @@ const ColorPicker = (
 ) => {
 	const {
 		enableAlpha = false,
-		color: colorProp,
+		color,
 		onChange,
-		defaultValue,
+		defaultValue = '#fff',
 		copyFormat,
 		...divProps
 	} = useContextSystem( props, 'ColorPicker' );
 
-	const [ color, setColor ] = useControlledValue( {
-		onChange,
-		value: colorProp,
-		defaultValue,
-	} );
```

Comment:
Hmm! I don't understand why we need to move away from `useControlledValue` in this case, but maybe this is an issue with the `useControlledValue` function itself.

I wonder if the fact that we're not memoizing the state updater here is causing any problem since the original updater from `useState` is supposed to be stable:
https://github.com/WordPress/gutenberg/blob/8ccd5e159c2ed90318601eef1c8e5e26d5cea76f/packages/components/src/utils/hooks/use-controlled-value.ts#L35-L38

Reply:
The only reason I removed the useControlledValue call was that everything seemed to be working as before without it. It seems on this case it is not needed.

Reply:
Just cc'ing @sarayourfriend who wrote this part of the `ColorPicker` component, just to make sure that removing `useControlledValue` was ok in this scenario

Reply:
The idea behind using `useControlledValue` was to allow the color picker to behave like a standard React input, that is, with or without a controlled external value.

```jsx
<ColorPicker onChange={setColor} />
```

Should work even though the `color` prop isn't being passed. Just like `<input onChange={setValue} />` works.

Reply:
This makes sense, thank you Sara!

@jorgefilipecosta , would it possible to add the `useControlledValue` hook back to the component?

Reply:
Hey @jorgefilipecosta , just wondering if you've had any time to work on this, or if you plan to anytime soon? Thank you!

Reply:
@ciampo and @jorgefilipecosta, I was investigating an issue with the color picker and gradients (reported in #36899) and it looks like re-instating `useControlledValue` and the `debounce` call resolves the issue, so I've opened up a PR in #36941 if either or you get the chance to take a look. I'm not very familiar with the `ColorPicker` component, so it can probably use some 👀 from folks who've worked with it before 🙂

Reply:
Thank you @andrewserong , I'll definitely have a look